mymake.cpp

```diff
@@ -75,7 +75,7 @@ void set_linux() {
 
 void set_mac() {
   preprocessor = "g++ -E";
-  compiler = "g++ -march=native -W -Wall -Wextra -pedantic -Wno-unused-parameter -Wno-implicit-fallthrough -c";
+  compiler = "g++ -march=native -W -Wall -Wextra -pedantic -Wno-unused-parameter -Wno-implicit-fallthrough -Wno-invalid-offsetof -c";
```

Comment:
Off-topic: I notice a bunch of new `-Winconsistent-missing-override`, `-Woverloaded-virtual`, and `-Wextra-semi` warnings locally on OSX. We should probably fix those somewhere. :)